.setup/install_system.sh

```diff
@@ -378,6 +378,13 @@ EOF
     chmod 0640 /etc/apache2/suexec/www-data
 
 
+	#############
+	#NPM install
+	#############
+
+	curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+	sudo apt-get install -y nodejs
```

Comment:
you don't need sudo, the script has to be run as root.

Though, please convert the tabs to spaces.